src/uwtools/set_config.py

```diff
@@ -39,50 +39,73 @@ def parse_args(argv):
         help='Path to a config base file.',
         required=True,
         type=path_if_file_exists,
-        )
+    )
+
+    parser.add_argument(
+        '--in_file_type',
+        help='Optional input file type',
+    )
```

Comment:
These three args should have some options for what the user can choose, recalling that we can't parse an input Field Table.

Reply:
I was going to put that into --values_needed, but you're right, the help description would be better

Reply:
Also removing the type overrides as this is fixed by the other fix